presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -121,6 +121,7 @@
 import com.facebook.presto.sql.tree.QualifiedName;
 import com.facebook.presto.type.BigintOperators;
 import com.facebook.presto.type.BooleanOperators;
+import com.facebook.presto.type.CharOperators;
```

Comment:
equality is for chars with the same length. Nonequality is for chars with different length. Updated commit